cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Install locally with pip
+
+```
+pip install cloudcomposerdiff
```

Comment:
```suggestion
cd composer-utilities
pip install cloudcomposerdiff
```

I'm assuming the user wants to be in the root directory of the utils

Reply:
Yes, that's expected.

I assume that users are in the cloudcomposerdiff directory when running the pip install command. 

I have extended the README to make this clearer.